doc/api-extensions.md

```diff
@@ -2129,3 +2129,7 @@ projects that are referencing that network via one of their forward zones.
 
 Existing projects that have `features.networks=true` will have `features.networks.zones=true` set automatically,
 but new projects will need to specify this explicitly.
+
+## `network_txqueuelen`
```

Comment:
I would suggest calling this `instance_nic_txqueuelength` to differentiate from network settings (as these are instance NIC settings) and avoiding arbitrarily cutting words when they don't need to be.

Reply:
Done